### DISCUSSION TITLE
How can I do D3.js in WebView?

### DISCUSSION THREAD

**Discussion by visualizor** [picoe/Eto#2162] [Q&A]
As title implies, I'd like to draw graphs and charts with D3.js library. Can that be done in WebView?
Just as a test, I copied the code from this repo to an HTML and put all the js in a script tag.
https://github.com/kriscfoster/d3-barchart
But nothing shows up in the `WebView`. Where should I start to look as debugging?

**Comment by greatawesome** [comment]
Does it need to be that particular library, or could you use other charting controls?

**Reply by visualizor** [reply]
no doesn't have to be d3, although i'm curious to learn what differences there are to make some more compatible (with eto webview) than others.

**Reply by greatawesome** [reply]
If it doesn't need to be in a Webview, there are a couple charting libraries that work in eto. Oxyplot, ScottPlot, and ZedGraph all have Eto versions.

If it does need to be in Webview, I won't be much help as I haven't used the Eto Webview control.
